src/mpyl/reporting/targets/jira.py

```diff
@@ -89,7 +89,7 @@ def from_config(config: dict):
         if not jira_config:
             raise KeyError('jira section needs to be defined in mpyl_config.yml')
         return JiraConfig(site=jira_config['site'], user_name=jira_config['userName'], password=jira_config['password'],
-                          ticket_pattern=re.compile(jira_config.get('ticketPattern', '[A-Za-z]{3,}-\\d+')),
+                          ticket_pattern=re.compile(jira_config.get('ticketPattern', '[A-Za-z]{2,}-\\d+')),
```

Comment:
This is just a fallback for when no pattern is configured. For the monorepo the pattern is configured here https://github.com/Vandebron/Vandebron/blob/c2e87c320916d74931639b1166f52289fd1a256f/mpyl_config.yml#L20.

Reply:
Did: https://github.com/Vandebron/Vandebron/pull/10495 :)_